tests/verifier.rs

```diff
@@ -178,8 +178,8 @@ fn test_verifier_err_lddw_cannot_be_last() {
 
 #[test]
 fn test_verifier_err_invalid_reg_dst() {
-    // r11 is disabled when sbpf_version.dynamic_stack_frames()=false, and only sub and add are
-    // allowed when sbpf_version.dynamic_stack_frames()=true
+    // r11 is disabled when sbpf_version.manual_stack_frame_bump()=false, and only sub and add are
+    // allowed when sbpf_version.manual_stack_frame_bump()=true
```

Comment:
r11 is not allowed for any version, is it?

Reply:
Should be r10